_docs/guides/bookinfo.md

```diff
@@ -58,9 +63,7 @@ and outgoing calls for the services, providing the hooks needed to externally co
 via the Istio control plane, routing, telemetry collection, and policy enforcement
 for the application as a whole.
 
-Before you begin, if you haven't already done so, setup Istio by following the instructions
-corresponding to your platform [installation guide]({{home}}/docs/setup/) and then
-run the application using the instructions in one of the following sections.
+To start the application, follow the instructions, below, corresponding to your Istio runtime environment.
```

Comment:
no commas after the "follow the instructions,.. below,...corresponding.." .. Basically the last two commas seem out of place, but take it with a grain of salt please :)..